src/java/fr/paris/lutece/plugins/blog/service/BlogParameterService.java

```diff
@@ -11,7 +16,12 @@ public class BlogParameterService
     private static BlogParameterService _singleton = new BlogParameterService( );
 
     public static final String DSKEY_DEFAULT_NUMBER_MANDATORY_TAGS = "blog.advanced_parameters.number_mandatory_tags";
+    public static final String DSKEY_DEFAULT_DATE_END_PUBLISHING = "blog.advanced_parameters.default_date_end_publishing";
     public static final String MARK_DEFAULT_NUMBER_MANDATORY_TAGS = "number_mandatory_tags";
+    public static final String MARK_DEFAULT_DATE_END_PUBLISHING = "default_date_end_publishing";
+
+    private static final String FIELD_DATEFORMAT = "yyyy-MM-dd";
+    private static final String DB_DATEFORMAT = "dd/MM/yyyy";
```

Comment:
Why using  a french date format to store the date ?

Why not just use "null" as max date ?